src/toggle/util/ToggleContext.tsx

```diff
@@ -0,0 +1,24 @@
+import {createContext, useContext} from "react";
+
+type ToggleContextValue = {
+  selectedToggleItemsState: string[];
+  onToggleItem: (dataId: string) => void;
+};
+
+const ToggleContext = createContext<ToggleContextValue>({
+  selectedToggleItemsState: [] as string[],
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
+  onToggleItem: () => {}
+});
+
```

Comment:
Let's add a displayName for the ToggleContext
```suggestion
ToggleContext.displayName = "ToggleContext";
```